azure/template.json

```diff
@@ -117,10 +117,10 @@
         "description": "Connection string for Sentry monitoring for backend app."
       }
     },
-    "sentryDSNContainers": {
-      "type": "array",
-      "metadata": {
-        "description": "Connection string for Sentry monitoring for background geocoding and other container instances."
+    "sentryDSNSideKiq":{
+      "type" : "string",
+      "metadata":{
+        "description" : "Sentry DSN for monitoring the background sidekiq container."
       }
```

Comment:
Should this be `sentryDSNBGJobs` maybe? i.e. `bg jobs` instead of `sidekiq`?

Reply:
Ya, I've just noticed that elsewhere in this file, we refer to a "background" container. Feels like that name should be aligned with the above parameter.

Reply:
done..